accepted/future-releases/0546-patterns/feature-specification.md

```diff
@@ -2748,20 +2748,30 @@ To match a pattern `p` against a value `v`:
         1.  Evaluate the key `expression` to `k` and call `containsKey(k)` on
             the value. If this returns `false`, the map does not match.
 
-        2.  Otherwise, evaluate `v[k]` and match the resulting value against
-            this entry's value subpattern. If it does not match, the map does
-            not match.
+        2.  Otherwise, evaluate `v[k]` to `r`.
+
+        3.  If `V` is non-nullable and `r` is `null` then the map does not
```

Comment:
This isn't enough, I think, because `V` may be neither non-nullable nor nullable (it may be unknown).

Reply:
You're right. Fixed.